server/utils/articles/deleteImage.util.js

```diff
@@ -0,0 +1,21 @@
+import cloudinary from 'cloudinary';
+import db from '../../models';
+
+const { Article } = db;
+
+const deleteImage = async (req, res, next) => {
+  const { slug } = req.params;
+  const article = await Article.findOne({
+    attributes: ['image'],
+    where: {
+      slug,
+    }
+  });
+  const imageUrl = article.dataValues.image;
+  if (!imageUrl) return next();
+  const secureImageId = imageUrl.slice(70, 90);
+  cloudinary.v2.uploader.destroy(secureImageId);
+  return next();
+};
```

Comment:
A little clarification please. Why 70 and 90?

Reply:
I had to splice the image url in the db to get the image id which is on cloudinary. 70 and 90 are the start and end of the string I need.
This is what is deleted from cloudinary

Reply:
Got it.